src/components/AnnouncementCard/AnnouncementCard.css

```diff
@@ -0,0 +1,43 @@
+.announcementCard{
+    box-shadow: rgba(0, 0, 0, 0.25) 0px 1px 4px;
+    margin: 20px 0px 15px 20px;
+    width: 400px;
+    padding: 15px 20px;
+  }
+  .header h2, h3{
+      display: inline-block;
+  }
+  .announcement{
+    font-size: 20px;
+    float: left;
+  }
+  .date{
+    font-size: 15px;
+    margin: 0;
+    float: right;
+    color: gray;
+  }
+  .description{
+    font-size: 15px;
+    padding-top: 10px;
+    margin: 0; 
+  }
+  .readmore{
+    color: #6666FF;
+    display: flex;
+    align-items: center;
+    padding-left: 120px;
+  }
+  .readmore h3{
+    font-size: 15px;
+    margin: 0;
+    font-weight: lighter;
+    letter-spacing: 0.5px;
+  }
+  .arrow {
+    border: solid #6666FF;
+    border-width: 0 3px 3px 0;
+    padding: 5px;
+    transform: rotate(-45deg);
+    -webkit-transform: rotate(-45deg);
+  }
```

Comment:
Can you delete this file? I don't think it's doing anything because you used Bootstrap